### DISCUSSION TITLE
Bug di Chia e il caricamento del portafoglio. Wallet si blocca  versione 1.1.5

### DISCUSSION THREAD

**Discussion by SeppiaOmbra** [Chia-Network/chia-blockchain#4613] [Support]
Impossibile connettersi al portafoglio. 

Impossibile connettersi dopo un arresto anomalo della macchina. Nessuna connessione al Wallet ne disconnessione in uscita solo schermo nero.

Ho provato **più volte** a:

cancellare la app da pc e tutti i file di sistema, nascosti e chiavi di registro.

Verificare l'apertura delle specifiche porte relative al wallet.

re installare la app da sito. far partire e rimanere anche più di un ora nella finestra

Ho seguito anche https://github.com/Chia-Network/chia-blockchain/issues/863

Ma proprio non ne vuole sapere di partire.

Ne capita sempre una!

Desktop (completare le seguenti informazioni):

Sistema operativo Windows 10 Pro
PROCESSORE: Intel(R) Core(TM) i9-10850K

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
